picard/ui/metadatabox.py

```diff
@@ -624,6 +624,8 @@ def _update_tags(self, new_selection=True, drop_album_caches=False):
 
             tag_diff.add('~length', str(orig_metadata.length), str(new_metadata.length),
                          removable=False, readonly=True)
+            if (len(files) == 1) and (settings['rename_files'] or settings['move_files']):
```

Comment:
Minor, but I'd remove the round brackets around `len(files) == 1`. Not really needed here.

Reply:
Okay!